tests/unit_tests/blas/batch/gemm_batch_stride.cpp

```diff
@@ -200,11 +211,14 @@ int test(device *dev, oneapi::mkl::layout layout, int64_t batch_size) {
     }
 
     // Compare the results of reference implementation and DPC++ implementation.
+    constexpr int tol_scalar = std::is_same_v<Ta, Ts> ? 10 : 40;
```

Comment:
When exactly do you need this lower tolerance? I would have expected `is_same_v<Tc, Ts>` because internal accumulation is done in `Tc` precision but I may be wrong.

Reply:
Although the computation is happening in fp32 according the compute type, I am still seeing differences in the output. They are small, I suspect it could be down the implementation and hardware differences. It is a larger discussion about how precise these computations must be to the reference, which is doing something slightly different due to the difference in input type.

Reply:
Agreed, this tolerance is fine for this PR.